test/simulator/timestamp/src/timestamp_manager.cpp

```diff
@@ -113,10 +113,12 @@ timestamp_manager::parse_config(const std::string &config,
      */
     while (std::getline(conf, token, ',')) {
         int pos = token.find('=');
-        if (pos == -1)
-            config_map.insert({trim(token), ""});
-        else
-            config_map.insert({trim(token.substr(0, pos)), trim(token.substr(pos + 1))});
+        if (token != "") {
```

Comment:
Could you explain what is being done here?

Reply:
In a case of a config that has a leading comma ie ",commit_timestamp=1" an empty string is passed to this function that we then need to ignore. I'll add a comment.

Reply:
Hmm, makes sense, thanks.